src/extractor/guidance/roundabout_handler.cpp

```diff
@@ -111,32 +111,57 @@ void RoundaboutHandler::invalidateExitAgainstDirection(const NodeID from_nid,
     if (in_edge_data.roundabout || in_edge_data.circular)
         return;
 
-    bool past_roundabout_angle = false;
-    const bool lhs = profile_properties.left_hand_driving;
-    const int step = lhs ? -1 : 1;
-    for (std::size_t cnt = 0, idx = lhs ? intersection.size() - 1 : 0; cnt < intersection.size();
-         ++cnt, idx += step)
+    std::cout << "invalidateExitAgainstDirection\n";
+
+    // Find range in which exits that must be invalidated (shaded areas):
+    // exit..end   exit..end  begin..exit for ↺ roundabouts
+    // ⭦           ⭦  ⭩         ⭦
+    //  ⭡⭧          ⭡⭩ ▒       ▒⭨⭡
+    //  ⭡⭦          ⭡⭨▒▒       ▒▒⭡⭨
+    // ⭧▒▒⭦        ⭧▒▒▒▒      ▒▒⭧
```

Comment:
It would be great if we could stay on ASCII based characters since in some browsers/OS this diff looks like this:

![screenshot from 2017-05-02 09-37-17](https://cloud.githubusercontent.com/assets/606968/25612773/79e5fe6e-2f1b-11e7-845d-0ca3d6e653d2.png)